hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncHBaseAdmin.java

```diff
@@ -3381,7 +3364,7 @@ private void getProcedureResult(long procId, CompletableFuture<Void> future, int
       .call();
   }
 
-  private <T> ServerRequestCallerBuilder<T> newServerCaller() {
+  <T> ServerRequestCallerBuilder<T> newServerCaller() {
     return this.connection.callerFactory.<T> serverRequest()
       .rpcTimeout(rpcTimeoutNs, TimeUnit.NANOSECONDS)
       .operationTimeout(operationTimeoutNs, TimeUnit.NANOSECONDS)
```

Comment:
An hbase3 client will not be able to talk to an hbase2 cluster, right? Or will it?

Reply:
This patch does not introduce any incompatible changes between client and server. And for the general question on whether hbase3 client can talk with hbase2 cluster, I do not know... We need to test it, I think the basic operation such as read and write could work, but the admin operations may be broken?

Reply:
File an issue? Would be sweet if the h3 client worked against h2.

Reply:
I mean, file an issue to investigate and fix andy teething problems that might be in the way of h3 working against an h2 cluster.

Reply:
I think we'd better do this after we cut branch-3? Otherwise the issue may be forgotten...